bitnami/redis/Chart.yaml

```
@@ -36,4 +36,4 @@ maintainers:
 name: redis
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/redis
-version: 20.0.3
+version: 20.0.4
```

```suggestion
version: 20.1.0
```

Please bump the version to a new minor one when adding new parameters to the chart